src/figuresClasses.ts

```diff
@@ -1,19 +1,113 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape : 'triangle' | 'circle' | 'rectangle';
+  color : Color;
+  getArea() : number;
 }
 
-export class Triangle {
+let square : number;
```

Comment:
you don't need this variable

Reply:
I think need this value becouse i have new helper function getRounding. 
I can do this if I don't use the getRouding helper function, but just return the formula without using variable square.